ai-data/llm-inference/reference-content/llama-2-70b.mdx

```diff
@@ -0,0 +1,87 @@
+---
+meta:
+  title: Understanding the Llama-2-70b-chat model
+  description: This page provides information on the Llama-2-70b-chat model
+content:
+  h1: Understanding the Llama-2-70b-chat model
+  paragraph: This page provides information on the Llama-2-70b-chat model
+tags: 
+categories:
+  - ia
+---
+
+
+
+## Model overview
+
+| Attribute       | Details                            |
+|-----------------|------------------------------------|
+| Provider        | [Meta](https://llama.meta.com/llama2)  |
+| Model Name      | `llama-2-70b-chat`                 |
+| Compatible Instances | H100 (FP8) - H100-2 (FP16)    |
+
+## Model names
+
+```bash
+meta/llama-2-70b-chat:fp8
+meta/llama-2-70b-chat:fp16
+```
+
+## Compatible Instances
+
+- [H100 (FP8)](https://www.scaleway.com/en/h100-pcie-try-it-now/)
+- [H100-2 (FP16)](https://www.scaleway.com/en/h100-pcie-try-it-now/)
+
+## Model introduction
+
+The Llama-2-70b-chat model, developed by Meta, is designed for various chat applications and customer service platforms. 
+Trained on diverse conversational data, it generates human-like responses and engages in meaningful dialogues.
+Its versatility makes it suitable for businesses seeking to enhance their customer interactions.
+
+## Why you will love it
+
+Llama-2-70b-chat offers seamless integration with chat applications and customer service platforms, facilitating smooth communication between businesses and their customers.
+Its robust performance and natural language understanding enhance user experience and foster customer satisfaction.
+
+## How to use it
+
+### Sending Inference requests
+
+To perform inference tasks with your Llama-2 deployed at Scaleway, use the following command:
+
+```bash
+curl -s \
+-H "X-Auth-Token: <IAM token>" \
+-H "Content-Type: application/json" \
+--request POST \
+--url "https://<Deployment UUID>.ifr.fr-par.scw.cloud/" \
+--data '{"text_input": "[INST]How can I use large language models to improve my customer service? [/INST]", "max_tokens": 200, "temperature": 0.2, "random_seed": 1, "top_p": 0.9}' | jq -r .text_output
+```
+
+Make sure to replace <IAM token> and <Deployment UUID> with your actual IAM token and the Deployment UUID you are targeting.
```

Comment:
```suggestion
Make sure to replace <IAM token> and <Deployment UUID> with your actual IAM token and the Deployment UUID you are targeting.
```
what is an IAM token? is it the API key? Best to replace it so we keep the names ISO all across the docs

Reply:
thank you @ldecarvalho-doc ! Replaced token by API key and linked to IAM API key how to.